src/update_upgrade/update-js10.2.0.sh

```diff
@@ -230,8 +226,10 @@ update_overcloud() {
 
     # Addition in 10.2.0: "openstack overcloud deploy" now requires --stack attribute or it will default to "overcloud"
     # Replace all "~" with value of $HOME when building command otherwise ~ will not be evaluated correctly.
+    
     info "Update overcloud plan next, additional -e args that will be applied: ${envs[@]//\~/$HOME}"
-    openstack overcloud deploy --debug --update-plan-only --stack $STACK_NAME --templates ~/pilot/templates/overcloud -e ~/pilot/templates/overcloud/overcloud-resource-registry-puppet.yaml ${envs[@]//\~/$HOME}
+
+    openstack overcloud deploy --update-plan-only  --debug  --log-file ~/update/overcloud_deploy_update_plan_only.log  -t 120  --stack red --templates ~/pilot/templates/overcloud -r ~/pilot/templates/roles_data.yaml ${envs[@]//\~/$HOME}
 
```

Comment:
--stack red , should be --stack $STACK_NAME ?

Reply:
Good catch guys thanks, left over from testing